A3AE/addons/templates/Templates/#Blueprints/Reb_Faction_Blueprint.sqf

```diff
@@ -0,0 +1,161 @@
+///////////////////////////
+//   Rebel Information   //
+///////////////////////////
+
+["name", ""] call _fnc_saveToTemplate;    //this line determines the faction name -- Example: ["name", "NATO"] - ENTER ONLY ONE OPTION
+
+["flag", ""] call _fnc_saveToTemplate;    //this line determines the flag -- Example: ["flag", "Flag_NATO_F"] - ENTER ONLY ONE OPTION
+["flagTexture", ""] call _fnc_saveToTemplate;    //this line determines the flag texture -- Example: ["flagTexture", "\A3\Data_F\Flags\Flag_NATO_CO.paa"] - ENTER ONLY ONE OPTION
+["flagMarkerType", ""] call _fnc_saveToTemplate;    //this line determines the flag marker type -- Example: ["flagMarkerType", "flag_NATO"] - ENTER ONLY ONE OPTION
+
+["vehiclesBasic", []] call _fnc_saveToTemplate;    //this line determines basic vehicles, the lightest kind available. -- Example: ["B_Quadbike_01_F"] -- Array, can have multiple entries
+["vehiclesLightUnarmed", []] call _fnc_saveToTemplate;    //this line determines light and unarmed vehicles. -- Example: ["B_MRAP_01_F"] -- Array, can have multiple entries
+["vehiclesLightArmed", []] call _fnc_saveToTemplate;    //this line determines light and armed vehicles -- Example: ["B_MRAP_01_hmg_F","B_MRAP_01_gmg_F"] -- Array, can have multiple entries
+["vehiclesTruck", []] call _fnc_saveToTemplate;    //this line determines the trucks -- Example: ["B_Truck_01_transport_F","B_Truck_01_covered_F"] -- Array, can have multiple entries
+["vehiclesAT", []] call _fnc_saveToTemplate;    //this line determines AT vehicle -- Example: ["B_Truck_01_transport_F","B_Truck_01_covered_F"] -- Array, can have multiple entries
+private _vehicleAA = [];    //this line determines AA vehicle -- Example: private _vehicleAA = [["B_Truck_01_transport_F","B_Truck_01_covered_F"] -- Array, can have multiple entries
```

Comment:
Clear out the weird syntax here. It's unlikely that people would care about DLC variance in their own template.

Reply:
please elaborate

Reply:
Do you want me to remove the `private _vehicleAA = []; ` and the `["vehiclesAA", _vehicleAA] call _fnc_saveToTemplate;` later on and just slap in a `["vehiclesAA", []] call _fnc_saveToTemplate;`?